### ISSUE TITLE
Cannot Fill Water Containers

### ISSUE THREAD

**Issue by MonaLux97** [jhaakma/ashfall#2]
I have followed the troubleshooting suggestions, so I believe my MWSE is up to date. I use Mod Organizer 2. I have ruled out mod conflicts by running only Ashfall & it's requirements in a new save. 

When I approach a water source such as a well or an open body of water, I can activate the menu. I can drink the water, but even if I have containers in my inventory (tested with combinations of bottles, flasks, pitchers, ext) the menu tells me I have no containers to fill. This also happens when I try to use the vendor fill option. I created the MWSE log, and have uploaded it here.

Thank you for your time, overwise I was having a really fun time with the mod. 
[MWSE.log](https://github.com/jhaakma/ashfall/files/5861294/MWSE.log)

**Comment by MonaLux97** [comment]
Ok so, I am the problem but I'm going to detail how I fixed the problem for other idiots. I suspected that maybe my MWSE was just not up to date despite having followed the directions, so I downloaded the newest file from the MWSE Nightly site, and replaced both my MWSE and LUA files manually. And magically the water problem was fixed, along with a dozen other little issues I'd noticed. 
So for anyone else using MO...do it by hand. Follow the installation steps but replace files manually.
My bad. Although I did sort of figure it was my bad, I just couldn't pinpoint *what*

**Comment by jhaakma** [comment]
Hey sorry I didn't get time to help you with this. There is an issue with old versions of MWSE where it couldn't connect to Github, and requires a manual install as you did here. Now that you've done that, the updater should work as normal. I did suspect this but when I looked at your mwse.log it said it was using the Jan 21 version of MWSE, which should be the latest version. 

Anyway I'm glad you figured it out!
